src/vs/workbench/contrib/notebook/browser/viewModel/notebookOutlineEntryFactory.ts

```diff
@@ -87,10 +87,8 @@ export class NotebookOutlineEntryFactory {
 				// Gathering symbols from the model is an async operation, but this provider is syncronous.
 				// So symbols need to be precached before this function is called to get the full list.
 				if (cachedEntries) {
-					// push code cell that is a parent of cached symbols if we are targeting the outlinePane
-					if (target === OutlineTarget.OutlinePane) {
-						entries.push(new OutlineEntry(index++, NotebookOutlineConstants.NonHeaderOutlineLevel, cell, preview, !!exeState, exeState ? exeState.isPaused : false));
-					}
+					// push code cell entry that is a parent of cached symbols, always necessary. filtering done elsewhere.
+					entries.push(new OutlineEntry(index++, NotebookOutlineConstants.NonHeaderOutlineLevel, cell, preview, !!exeState, exeState ? exeState.isPaused : false));
```

Comment:
@Yoyokrazy 
Please do ensure we clean this in next iteration, 
I understand this is a quick fix for recovery, but the code is weird, as we're passing in the `target` as an argument into the method.

Reply:
Yep, this will get addressed in the restructuring that we discussed after the notebook sync. It'll need work after the structure of the `notebookOutlineProvider` changes to reflect the rawModel vs viewModel ideas we have talked about.